app/index.js

```diff
@@ -1,34 +1,33 @@
-const { setup } = require('./insights.js')
-require('log-timestamp')
-const messaging = require('./messaging/index.js')
-const { createServer } = require('./server.js')
-
-let serverLogger
+import { setup } from './insights.js'
+import 'log-timestamp'
+import { startSfdMessageReceiver, stopSfdMessageReceiver } from './messaging/index.js'
+import server from './server.js'
 
 const init = async () => {
-  const server = await createServer()
+  setup(server.logger)
```

Comment:
I honestly don't know but was intending to dig in and try to find out. Was there a reason why insights was initialised before the server was started?

Reply:
Yep no idea on this - I guess it would make sense to initialise them after?